pkg/config/system_probe.go

```diff
@@ -123,8 +123,8 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.enable_goroutine_stacktraces"), false)
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.delta_profiles"), true)
 	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.custom_attributes"), []string{"module"})
-	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.unix_socket"), "")
-	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.extra_tags"), []string{})
+	cfg.BindEnvAndSetDefault(join(spNS, "internal_profiling.unix_socket"), "", "DD_SYSTEM_PROBE_INTERNAL_PROFILING_UNIX_SOCKET")
```

Comment:
here the env var would be `DD_SYSTEM_PROBE_CONFIG_INTERNAL_PROFILING_UNIX_SOCKET`